### DISCUSSION TITLE
AWS Amplify

### DISCUSSION THREAD

**Discussion by orioletech** [vercel/next.js#36683] [Help]
I'm trying to host a next js repo on Vercel however because it utilises AWS amplify it wont work. Talking directly to Vercel support I was directed to open a discussion here to see if anyone had a solution. 

Any support with this would be amazing!

**Comment by leerob** [comment]
Hey! We don't have the ability to update/modify the Amplify docs, but it would be great if it mentioned the versions of Next.js supported here: https://docs.amplify.aws/guides/hosting/nextjs/q/platform/js/#getting-started

It sounds like they don't support Next.js yet: https://github.com/aws-amplify/amplify-js/issues/9145#issuecomment-1083459532

You can open an issue here to chat with their team: https://github.com/aws-amplify/amplify-js/issues?q=is%3Aissue+is%3Aopen+next.js
